src/app/threat-dashboard/threat-dashboard.component.html

```diff
@@ -1,25 +1,65 @@
 <div [@simpleFadeIn] class="container-fluid" *ngIf="!loading; else loadingBlock">
+
     <div class="row flex">
-        <div class="side-panel" [@inOutAnimation] *ngIf="minimize !== true; else minimizedBlock" (mouseover)="showMinimizeBtn = true"
-            (mouseleave)="showMinimizeBtn = false">
-            <div class="minimizeControls" [style.display]="showMinimizeBtn ? 'block': 'none'">
-                <button mat-icon-button (click)="minimize = true" matTooltip="Minimize" matTooltipPosition="right">
-                    <mat-icon class="mat-24" aria-hidden="true" aria-label="Minimize side panel" #minimizeBtn>chevron_left</mat-icon>
-                </button>
-            </div>
-            <unf-side-panel [threatReport]="threatReport" [attackPatterns]="attackPatterns" (modifiedBoundries)="onBoundriesModified($event)"></unf-side-panel>
-        </div>
+
+        <unfetter-side-panel [item]="threatReport" [collapsible]="false">
+
+            <sidepanel-option-item label="Edit" icon="mode_edit" [disabled]="true"></sidepanel-option-item>
+            <sidepanel-option-item label="Share" [icon]="'share'" [disabled]="true"></sidepanel-option-item>
+            <sidepanel-option-item label="Delete" [icon]="'delete'"
+                    (click)="onDeleteExternalRef($event)"></sidepanel-option-item>
+
+            <sidepanel-master-list-trigger (click)="showMasterList($event)"></sidepanel-master-list-trigger>
+
+            <sidepanel-list-item label="Threat Reports" icon="assets/icon/stix-icons/svg/all-defs.svg#campaign"
+                    [items]="threatReport.reports" [expandable]="true">
+                <mat-list-item *ngFor="let item of threatReport.reports; trackBy:trackByFn"
```

Comment:
trackByFn is missing in corresponding component, port over from original side panel